number_parser/parser.py

```diff
@@ -114,15 +116,17 @@ def tokenize(input_string):
     tokens = re.split(r'(\W)', input_string)
```

Comment:
If not doing anything with variable, we can directly `return re.split(r'(\W)', input_string)`